automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -171,6 +171,10 @@ object Leonardo extends RestClient with LazyLogging {
       handleClusterResponse(patchRequest(url + path, clusterRequest))
     }
   }
+
+  object app {
```

Comment:
what is this for?

Reply:
Didn't realize this part made it into the PR. This is for the automation ticket. Will remove this